### ISSUE TITLE
val acc 관련 문제 

### ISSUE THREAD

**Issue by seonghoonham97** [Korea-Health-Datathon/KHD2020#60]
train accuracy, train loss, val loss 전부 제대로 학습이 되는 방향으로 값들이 변화하는데, val acc만 첫번째 값 그대로 한번도 변화하지 않습니다.  혹시 이럴때 어떤 가능성이 있나요????

**Comment by gusdkgusdk** [comment]
혹시 어떤 트랙이신가요?

**Comment by seonghoonham97** [comment]
A track 입니다. 제공된 base line code로 돌렸을 때도 동일하게 초기 val acc 값으로 계속 print 됩니다.

**Comment by seonghoonham97** [comment]
summit을 했을때 test set에 대해서는 문제없이 성능이 향상되고, val loss는 하락하는 것을 보아, 모델 학습은 되는것 같습니다.

**Comment by gusdkgusdk** [comment]
validation set에 레이블이 고르게 분포되지 않을 경우 동일하게 나올 수 있습니다. validation set 생성 부분에 대한 코드를 수정하여 테스트해보시면 좋을 것 같습니다. 

submit은 잘된다니 다행이네요!
